### PR TITLE
add 4.15.0.Final and 12.15.0.GA to tools.jboss.org

### PR DESCRIPTION
**Pull request by sbouchet**
Signed-off-by: Stephane Bouchet <sbouchet@redhat.com>

### CODE REVIEW

**Comment by sbouchet** [comment]
@jeffmaury : blogs path to be modified when contents will be available.

File: _config/products.yml [hunk 1]
```diff
@@ -156,12 +164,38 @@ devstudio:
...
+      12.15.0.GA:
+        archived: false
+        update_site_url: https://devstudio.redhat.com/12/stable/updates/
+        marketplace_url: https://marketplace.eclipse.org/node/1616973
+        marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1616973
+        release_date: 2020-04-28
+        supported_devstudio_is_version: 12.11.0.GA
+        documentation_url: https://access.redhat.com/site/documentation/en-US/Red_Hat_JBoss_Developer_Studio/
+        release_notes_url: https://access.redhat.com/site/documentation/en-US/Red_Hat_JBoss_Developer_Studio/
+        blog_announcement_url: /tbd
```
Comment (jeffmaury) [inline _config/products.yml:180]: Change to /blog/12.15.0.ga.html

File: _config/products.yml [hunk 2]
```diff
@@ -1522,11 +1556,31 @@ jbt_core:
...
+    simrel-2020-06:
+      4.16.x.Nightly:
         update_site_url: http://download.jboss.org/jbosstools/photon/snapshots/updates/
-      4.15.0.AM1:
+    simrel-2020-03:
+      4.15.0.Final:
         archived: false
+        release_date: 2020-04-28
+        supported_jbt_is_version: 4.11.0.Final
+        blog_announcement_url: /tbd
```
Comment (jeffmaury) [inline _config/products.yml:1567]: Change to /blog/12.15.0.ga.html

Comment (sbouchet) [reply _config/products.yml]: done.

File: _config/products.yml (hunk 1 above)
Comment (sbouchet) [reply _config/products.yml]: done.

**Pull request merged**